.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>sdcs</name>
```

Comment:
Change the project name to com.musala.sdcs, so that it corresponds to the package names.